apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -144,7 +144,7 @@ def get_registrations():
                 registrations.city,
                 registrations.state,
                 registrations.zip
-                FROM registrations"""
+                FROM registrations {f'WHERE registration_id = {reg_id}' if reg_id is not None else ''}"""
```

Comment:
I think a cleaner way to do this would be that instead of an f string you can just do `registration_id  = %s` and then do `cur.execute(query, (reg_id))` (can also add the logic for reg_id somewhere before this). This would also keep it consistent with how other queries are structured as well

Reply:
Generally, we do this because it's safer and lets Django protect us from SQL injection, we should definitely do that.

Reply:
@fnets you're saying I should implement Shayan's idea?

Reply:
Clarified, will make this change!